packages/components/organisms/f-checkout/src/components/Guest.vue

```diff
@@ -103,7 +116,11 @@ export default {
     methods: {
         ...mapActions(VUEX_CHECKOUT_MODULE, [
             'updateCustomerDetails'
-        ])
+        ]),
+
+        onGuestFieldBlur (field) {
+            this.onFieldBlur(VALIDATIONS.guest, field);
+        }
```

Comment:
Do we need this? Can't we just call `onFieldBlur` directly similar to the other parts of the code?

Reply:
The problem is that you can't call `VALIDATIONS` inside the template unless you wrap it with a computed property